app/views/docs/command-line.phtml

```diff
@@ -163,7 +131,161 @@ brew install --HEAD appwrite</code></pre>
+            <td>string</td>
+            <td>Name of your Appwrite Project.</td>
+        </tr>
+        <tr>
+            <td><code>functions</code></td>
+            <td>Array of <a href="/docs/command-line#function-options">functions</a></td>
+            <td>Configuration of functions in your project.</td>
+        </tr>
+        <tr>
+            <td><code>collections</code></td>
+            <td>Array of <a href="/docs/command-line#collection-options">collections</a></td>
+            <td>Configuration of collections in your project.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/command-line#function-options" id="function-options">Function Options</a></h3>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 150px">Option</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>$id</td>
+            <td>string</td>
+            <td>Function ID.</td>
+        </tr>
+        <tr>
+            <td><code>name</td>
+            <td>string</td>
+            <td>Function Name.</td>
+        </tr>
+        <tr>
+            <td><code>runtime</td>
+            <td>string</td>
+            <td>Function runtime, must be enabled. <a href="docs/functions#supportedRuntimes">Learn more about runtimes</a>.</td>
+        </tr>
+        <tr>
+            <td><code>path</code></td>
+            <td>string</td>
+            <td>Path to folder containing the function's source code.</td>
+        </tr>
+        <tr>
+            <td><code>entrypoint</code></td>
+            <td>string</td>
+            <td>Entry point of the function relative to the folder specified in path.</td>
+        </tr>
+        <tr>
+            <td><code>ignore</code></td>
+            <td>Array of strings</td>
+            <td>Files to ignore inside the function. <a href="docs/functions#ignoreFiles">Learn more about ignoring files</a>.</td>
+        </tr>
+        <tr>
+            <td><code>execute</code></td>
+            <td>Array of strings</td>
+            <td>Grants execute permissions to users. <a href="docs/rest#permissions">Learn more about permission strings</a>.</td>
+        </tr>
+        <tr>
+            <td><code>events</code></td>
+            <td>Array of strings</td>
+            <td>Events that trigger the function to execute. <a href="docs/events">Learn more about events</a>.</td>
+        </tr>
+        <tr>
+            <td><code>schedule</code></td>
+            <td>string</td>
+            <td>Execution schedule for the function as a <a href="https://en.wikipedia.org/wiki/Cron" target="_blank" rel="noopener">cron expression</a>.</td>
+        </tr>
+        <tr>
+            <td><code>timeout</code></td>
+            <td>int</td>
+            <td>Execution timeout of the function in seconds, with a maximum configurable limit of 900 seconds.</td>
```

Comment:
the maximum limit depends on the `_APP_FUNCTIONS_TIMEOUT` env var

Reply:
We're gonna pretend this doesn't exist cause cloud :P